packages/tomls/src/token/tokens/dai.toml

```diff
@@ -40,20 +40,20 @@ func = "setFeatureFlagAllowAll"
 args = ["<%= keccak256(hexlify('global')) %>", true]
 depends = ["invoke.upgrade_dai_proxy"]
 
-# [invoke.dai_set_pausers]
-# target = ["DAIProxy"]
-# fromCall.func = "owner"
-# func = "setDeniers"
-# args = [
-#   "<%= keccak256(hexlify('global')) %>",
-#   [
-#     "<%= settings.pauser1 %>",
-#     "<%= settings.pauser2 %>",
-#     "<%= settings.pauser3 %>",
-#     "<%= settings.pauser4 %>",
-#   ],
-# ]
-# depends = ["invoke.upgrade_dai_proxy"]
+[invoke.dai_set_pausers]
```

Comment:
does this mean dai pausers were not set?

Reply:
yes

Reply:
shall we add tests for this?

Reply:
yes, what if we do this in a separate PR?

Reply:
since it's part of the bigger scope rather than testnet deployments?